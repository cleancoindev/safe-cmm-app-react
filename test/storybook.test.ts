// See https://storybook.js.org/docs/react/workflows/snapshot-testing for further details
import initStoryshots, {
  Stories2SnapsConverter,
} from "@storybook/addon-storyshots";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";

initStoryshots({
  asyncJest: true, // this is the option that activates the async behaviour
  test: ({
    story,
    context,
    done, // --> callback passed to test method when asyncJest option is true
  }) => {
    const converter = new Stories2SnapsConverter();
    const snapshotFilename = converter.getSnapshotFileName(context);
    const storyElement = story.render();
    if (storyElement === undefined) {
      console.log(`story`, story);
      console.log(`context`, context);
    }

    // mount the story
    const tree = mount(storyElement);

    const waitTime = 1;
    setTimeout(() => {
      if (snapshotFilename) {
        expect(toJson(tree.update())).toMatchSpecificSnapshot(snapshotFilename);
      }

      done();
    }, waitTime);
  },
});
