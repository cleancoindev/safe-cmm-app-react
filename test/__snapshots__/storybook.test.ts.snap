// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots basic/display/Message Default 1`] = `
<div
  className="Message__Wrapper-sc-11ggxo0-0 cueRSQ"
  label="This is an error"
  type="error"
>
  <div>
    <span
      className="sc-fujyUd cUegfu"
      color="error"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <rect
            className="icon-color"
            height="5"
            rx="1"
            width="2"
            x="7"
            y="4"
          />
          <path
            className="icon-color"
            d="M8 9.9c.608 0 1.1.492 1.1 1.1 0 .608-.492 1.1-1.1 1.1-.608 0-1.1-.492-1.1-1.1 0-.608.492-1.1 1.1-1.1z"
            fillRule="nonzero"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
    <p
      className="sc-pNWxx kqjPsd"
      size="lg"
    >
      This is an error
    </p>
  </div>
</div>
`;

exports[`Storyshots basic/display/Message Warning 1`] = `
<div
  className="Message__Wrapper-sc-11ggxo0-0 zwFwx"
  label="This is a warning"
  type="warning"
>
  <div>
    <span
      className="sc-fujyUd gaLBot"
      color="warning"
    >
      <svg
        height="16"
        viewBox="0 0 17 16"
        width="17"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 5.996c.553 0 1 .447 1 1v.998c0 .553-.447 1-1 1-.552 0-1-.447-1-1v-.998c0-.553.448-1 1-1M8 9.895c.607 0 1.101.492 1.101 1.1 0 .607-.494 1.1-1.1 1.1-.608 0-1.1-.493-1.1-1.1 0-.608.492-1.1 1.1-1.1"
          />
          <path
            className="icon-color"
            d="M8 0c-.383 0-.766.193-.975.581L.133 13.373c-.396.734.138 1.624.974 1.624h13.786c.836 0 1.37-.89.974-1.624L8.974.581C8.766.193 8.384 0 8 0m0 3l5.386 9.997H2.613L8 3"
          />
        </g>
      </svg>
    </span>
    <p
      className="sc-pNWxx kqjPsd"
      size="lg"
    >
      This is a warning
    </p>
  </div>
</div>
`;

exports[`Storyshots basic/display/Message With Component Children 1`] = `
<div
  className="Message__Wrapper-sc-11ggxo0-0 cueRSQ"
  label="This is an error"
  type="error"
>
  <div>
    <span
      className="sc-fujyUd cUegfu"
      color="error"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <rect
            className="icon-color"
            height="5"
            rx="1"
            width="2"
            x="7"
            y="4"
          />
          <path
            className="icon-color"
            d="M8 9.9c.608 0 1.1.492 1.1 1.1 0 .608-.492 1.1-1.1 1.1-.608 0-1.1-.492-1.1-1.1 0-.608.492-1.1 1.1-1.1z"
            fillRule="nonzero"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
    <p
      className="sc-pNWxx kqjPsd"
      size="lg"
    >
      This is an error
    </p>
  </div>
  <div
    className="msgBody"
  >
    <p
      className="sc-pNWxx cbKbyz"
      color="shadow"
      size="sm"
    >
      The specified
       
      <span
        className="sc-pNWxx hcuugb"
        size="sm"
      >
        Start Price
      </span>
       
      is at least 2% higher than the current indicated market price. If intentional continue with your order.
    </p>
  </div>
</div>
`;

exports[`Storyshots basic/display/Message With Text Children 1`] = `
<div
  className="Message__Wrapper-sc-11ggxo0-0 cueRSQ"
  label="This is an error"
  type="error"
>
  <div>
    <span
      className="sc-fujyUd cUegfu"
      color="error"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <rect
            className="icon-color"
            height="5"
            rx="1"
            width="2"
            x="7"
            y="4"
          />
          <path
            className="icon-color"
            d="M8 9.9c.608 0 1.1.492 1.1 1.1 0 .608-.492 1.1-1.1 1.1-.608 0-1.1-.492-1.1-1.1 0-.608.492-1.1 1.1-1.1z"
            fillRule="nonzero"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
    <p
      className="sc-pNWxx kqjPsd"
      size="lg"
    >
      This is an error
    </p>
  </div>
  <p
    className="sc-pNWxx gihEqi msgBody"
    color="shadow"
    size="md"
  >
    The specified Start Price is at least 2% higher than the current indicated market price. If intentional continue with your order.
  </p>
</div>
`;

exports[`Storyshots basic/display/SubtextAmount Amount Is A Component 1`] = `
<div
  className="SubtextAmount__Wrapper-sc-64axk8-0 dqYpmJ"
>
  <span
    className="sc-pNWxx kUltXX"
    color="shadow"
    size="md"
  >
    Something else:
  </span>
  <span
    className="sc-pNWxx fjXQaZ"
    size="md"
  >
    <span>
      1250 ETH
    </span>
  </span>
</div>
`;

exports[`Storyshots basic/display/SubtextAmount Default 1`] = `
<div
  className="SubtextAmount__Wrapper-sc-64axk8-0 dqYpmJ"
>
  <span
    className="sc-pNWxx kUltXX"
    color="shadow"
    size="md"
  >
    Short subtext:
  </span>
  <span
    className="sc-pNWxx fjXQaZ"
    size="md"
  >
    ~ $1,323,545.2
  </span>
</div>
`;

exports[`Storyshots basic/display/SubtextAmount Inline 1`] = `
<div
  className="SubtextAmount__Wrapper-sc-64axk8-0 jONWSl"
>
  <span
    className="sc-pNWxx kUltXX"
    color="shadow"
    size="md"
  >
    Short subtext:
  </span>
  <span
    className="sc-pNWxx fjXQaZ"
    size="md"
  >
    ~ $1,323,545.2
  </span>
</div>
`;

exports[`Storyshots basic/display/TextWithTooltip Component As Text 1`] = `
<span
  className="TextWithTooltip__Wrapper-fy5psz-0 bJaFOb"
>
  <span
    className="sc-pNWxx GzZgV"
    color="error"
    size="lg"
  >
    <span>
      I'm a component
    </span>
  </span>
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="This is a nice tooltip"
  >
    <span
      className="sc-fujyUd dFgrIu"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
          <rect
            className="icon-color"
            height="2"
            rx="1"
            width="2"
            x="7"
            y="10"
          />
          <path
            className="icon-color"
            d="M7.385 5.933c0-.3.259-.548.585-.548.325 0 .583.249.583.548.003.222-.026.266-.366.514l-.027.02c-.774.565-1.049 1.035-1.007 1.856v.098c-.001.453.365.819.817.819.451 0 .817-.366.817-.818v-.14c-.01-.228.005-.252.337-.495l.027-.02c.767-.56 1.05-1.03 1.037-1.847 0-1.196-.997-2.17-2.218-2.17-1.224 0-2.22.975-2.22 2.183 0 .451.366.817.818.817.451 0 .817-.366.817-.817z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
  </span>
</span>
`;

exports[`Storyshots basic/display/TextWithTooltip Deactivated 1`] = `
<span
  className="TextWithTooltip__Wrapper-fy5psz-0 bJaFOb"
>
  <span
    className="sc-pNWxx eFDhan"
    color="shadow"
    size="lg"
  >
    Label label
  </span>
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="This is a nice tooltip"
  >
    <span
      className="sc-fujyUd dFgrIu"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
          <rect
            className="icon-color"
            height="2"
            rx="1"
            width="2"
            x="7"
            y="10"
          />
          <path
            className="icon-color"
            d="M7.385 5.933c0-.3.259-.548.585-.548.325 0 .583.249.583.548.003.222-.026.266-.366.514l-.027.02c-.774.565-1.049 1.035-1.007 1.856v.098c-.001.453.365.819.817.819.451 0 .817-.366.817-.818v-.14c-.01-.228.005-.252.337-.495l.027-.02c.767-.56 1.05-1.03 1.037-1.847 0-1.196-.997-2.17-2.218-2.17-1.224 0-2.22.975-2.22 2.183 0 .451.366.817.818.817.451 0 .817-.366.817-.817z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
  </span>
</span>
`;

exports[`Storyshots basic/display/TextWithTooltip Default 1`] = `
<span
  className="TextWithTooltip__Wrapper-fy5psz-0 bJaFOb"
>
  <span
    className="sc-pNWxx kqjPsd"
    size="lg"
  >
    Label label
  </span>
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="This is a nice tooltip"
  >
    <span
      className="sc-fujyUd dFgrIu"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
          <rect
            className="icon-color"
            height="2"
            rx="1"
            width="2"
            x="7"
            y="10"
          />
          <path
            className="icon-color"
            d="M7.385 5.933c0-.3.259-.548.585-.548.325 0 .583.249.583.548.003.222-.026.266-.366.514l-.027.02c-.774.565-1.049 1.035-1.007 1.856v.098c-.001.453.365.819.817.819.451 0 .817-.366.817-.818v-.14c-.01-.228.005-.252.337-.495l.027-.02c.767-.56 1.05-1.03 1.037-1.847 0-1.196-.997-2.17-2.218-2.17-1.224 0-2.22.975-2.22 2.183 0 .451.366.817.818.817.451 0 .817-.366.817-.817z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
  </span>
</span>
`;

exports[`Storyshots basic/display/TextWithTooltip Error 1`] = `
<span
  className="TextWithTooltip__Wrapper-fy5psz-0 bJaFOb"
>
  <span
    className="sc-pNWxx GzZgV"
    color="error"
    size="lg"
  >
    Label label
  </span>
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="This is a nice tooltip"
  >
    <span
      className="sc-fujyUd dFgrIu"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
          <rect
            className="icon-color"
            height="2"
            rx="1"
            width="2"
            x="7"
            y="10"
          />
          <path
            className="icon-color"
            d="M7.385 5.933c0-.3.259-.548.585-.548.325 0 .583.249.583.548.003.222-.026.266-.366.514l-.027.02c-.774.565-1.049 1.035-1.007 1.856v.098c-.001.453.365.819.817.819.451 0 .817-.366.817-.818v-.14c-.01-.228.005-.252.337-.495l.027-.02c.767-.56 1.05-1.03 1.037-1.847 0-1.196-.997-2.17-2.218-2.17-1.224 0-2.22.975-2.22 2.183 0 .451.366.817.818.817.451 0 .817-.366.817-.817z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
  </span>
</span>
`;

exports[`Storyshots basic/display/TextWithTooltip Small 1`] = `
<span
  className="TextWithTooltip__Wrapper-fy5psz-0 bJaFOb"
>
  <span
    className="sc-pNWxx hcuugb"
    size="sm"
  >
    Label label
  </span>
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="This is a nice tooltip"
  >
    <span
      className="sc-fujyUd dFgrIu"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
          <rect
            className="icon-color"
            height="2"
            rx="1"
            width="2"
            x="7"
            y="10"
          />
          <path
            className="icon-color"
            d="M7.385 5.933c0-.3.259-.548.585-.548.325 0 .583.249.583.548.003.222-.026.266-.366.514l-.027.02c-.774.565-1.049 1.035-1.007 1.856v.098c-.001.453.365.819.817.819.451 0 .817-.366.817-.818v-.14c-.01-.228.005-.252.337-.495l.027-.02c.767-.56 1.05-1.03 1.037-1.847 0-1.196-.997-2.17-2.218-2.17-1.224 0-2.22.975-2.22 2.183 0 .451.366.817.818.817.451 0 .817-.366.817-.817z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
  </span>
</span>
`;

exports[`Storyshots basic/display/TextWithTooltip Warning 1`] = `
<span
  className="TextWithTooltip__Wrapper-fy5psz-0 bJaFOb"
>
  <span
    className="sc-pNWxx beZLye"
    color="warning"
    size="lg"
  >
    Label label
  </span>
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="This is a nice tooltip"
  >
    <span
      className="sc-fujyUd dFgrIu"
    >
      <svg
        height="16"
        viewBox="0 0 16 16"
        width="16"
        xmlns="http://www.w3.org/2000/svg"
      >
        <g
          fill="none"
          fillRule="evenodd"
        >
          <path
            d="M0 0H16V16H0z"
          />
          <path
            className="icon-color"
            d="M8 15c3.866 0 7-3.134 7-7s-3.134-7-7-7-7 3.134-7 7 3.134 7 7 7zm0-2c-2.761 0-5-2.239-5-5s2.239-5 5-5 5 2.239 5 5-2.239 5-5 5z"
            fillRule="nonzero"
          />
          <rect
            className="icon-color"
            height="2"
            rx="1"
            width="2"
            x="7"
            y="10"
          />
          <path
            className="icon-color"
            d="M7.385 5.933c0-.3.259-.548.585-.548.325 0 .583.249.583.548.003.222-.026.266-.366.514l-.027.02c-.774.565-1.049 1.035-1.007 1.856v.098c-.001.453.365.819.817.819.451 0 .817-.366.817-.818v-.14c-.01-.228.005-.252.337-.495l.027-.02c.767-.56 1.05-1.03 1.037-1.847 0-1.196-.997-2.17-2.218-2.17-1.224 0-2.22.975-2.22 2.183 0 .451.366.817.818.817.451 0 .817-.366.817-.817z"
            fillRule="nonzero"
          />
        </g>
      </svg>
    </span>
  </span>
</span>
`;

exports[`Storyshots basic/display/Tooltip Default 1`] = `
<div>
  This text has a tooltip
   
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="I am a tooltip!"
  >
    <span
      style={
        Object {
          "borderBottom": "1px dashed black",
        }
      }
    >
      here
    </span>
  </span>
</div>
`;

exports[`Storyshots basic/display/Tooltip Interactive 1`] = `
<div>
  This text has a tooltip
   
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title={null}
  >
    <span
      style={
        Object {
          "borderBottom": "1px dashed black",
        }
      }
    >
      here
    </span>
  </span>
</div>
`;

exports[`Storyshots basic/display/Tooltip Multi Line Tooltip 1`] = `
<div>
  This text has a tooltip
   
  <span
    aria-describedby={null}
    className=""
    onBlur={[Function]}
    onFocus={[Function]}
    onMouseLeave={[Function]}
    onMouseOver={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    title="Each bracket consists of a single Ethereum address that places a buy-low and a sell-high order. Everytime the price goes through a bracket and activates both orders, the CMM provider earns the spread"
  >
    <span
      style={
        Object {
          "borderBottom": "1px dashed black",
        }
      }
    >
      here
    </span>
  </span>
</div>
`;

exports[`Storyshots basic/input/BracketsInput Default 1`] = `
<form
  autoComplete="off"
  noValidate={true}
  onSubmit={[Function]}
>
  <div
    className="TextFieldWithCustomLabel__Wrapper-a48xik-0 jTvESA BracketsInput__Wrapper-o6iqk4-0 hXcsFe"
    onChange={[Function]}
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
    />
    <div
      checked={false}
      className="MuiFormControl-root MuiTextField-root sc-fKgIGh ibXjNK BracketsInput__Wrapper-o6iqk4-0 hXcsFe"
      min={1}
      step={1}
    >
      <div
        className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiFilledInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="number"
        />
      </div>
    </div>
  </div>
</form>
`;

exports[`Storyshots basic/input/BracketsInput Filled 1`] = `
<form
  autoComplete="off"
  noValidate={true}
  onSubmit={[Function]}
>
  <div
    className="TextFieldWithCustomLabel__Wrapper-a48xik-0 kHHKeH BracketsInput__Wrapper-o6iqk4-0 hXcsFe"
    onChange={[Function]}
    value="7"
    width={75}
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
    />
    <div
      checked={true}
      className="MuiFormControl-root MuiTextField-root sc-fKgIGh ibXjNK BracketsInput__Wrapper-o6iqk4-0 hXcsFe"
      min={1}
      step={1}
      width={75}
    >
      <div
        className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiFilledInput-input"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="number"
          value="7"
        />
      </div>
    </div>
  </div>
</form>
`;

exports[`Storyshots basic/input/ButtonLink Default 1`] = `
<form
  autoComplete="off"
  noValidate={true}
  onSubmit={[Function]}
>
  <button
    className="sc-dIvqjp kBehMs ButtonLink__Wrapper-k0qv6a-0 bbPfHr"
    color="primary"
    onClick={[Function]}
    type="button"
  >
    <p
      className="sc-pNWxx fjvhwv"
      color="primary"
      size="lg"
    >
      button
    </p>
  </button>
</form>
`;

exports[`Storyshots basic/input/ButtonLink Text Is A Number 1`] = `
<form
  autoComplete="off"
  noValidate={true}
  onSubmit={[Function]}
>
  <button
    className="sc-dIvqjp kBehMs ButtonLink__Wrapper-k0qv6a-0 bbPfHr"
    color="primary"
    onClick={[Function]}
    type="button"
  >
    <p
      className="sc-pNWxx fjvhwv"
      color="primary"
      size="lg"
    >
      56446.44
    </p>
  </button>
</form>
`;

exports[`Storyshots basic/input/Link Default 1`] = `
<a
  className="Link__StyledLink-sc-1fldheq-0 isSVSR"
  color="primary"
  href="http://going.somewhere"
  onClick={[Function]}
  rel="noopener noreferrer"
  target="_blank"
  type="button"
>
  <span
    className="sc-pNWxx fjvhwv"
    color="primary"
    size="lg"
  >
    link
  </span>
</a>
`;

exports[`Storyshots basic/input/Link Text Is A Number 1`] = `
<a
  className="Link__StyledLink-sc-1fldheq-0 isSVSR"
  color="primary"
  href="http://going.somewhere"
  onClick={[Function]}
  rel="noopener noreferrer"
  target="_blank"
  type="button"
>
  <span
    className="sc-pNWxx fjvhwv"
    color="primary"
    size="lg"
  >
    56446.44
  </span>
</a>
`;

exports[`Storyshots basic/input/NumberInput Custom Width 1`] = `
<form
  autoComplete="off"
  noValidate={true}
  onSubmit={[Function]}
>
  <div
    className="TextFieldWithCustomLabel__Wrapper-a48xik-0 kFdJRL"
    onChange={[Function]}
    width="120px"
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
    >
      <div>
        Label
      </div>
    </label>
    <div
      className="MuiFormControl-root MuiTextField-root sc-fKgIGh ibXjNK"
    >
      <div
        className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl MuiInputBase-adornedEnd MuiFilledInput-adornedEnd"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputAdornedEnd MuiFilledInput-inputAdornedEnd"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
        />
        <div
          className="MuiInputAdornment-root MuiInputAdornment-filled MuiInputAdornment-positionEnd"
        >
          <span>
            Bla
          </span>
        </div>
      </div>
    </div>
  </div>
</form>
`;

exports[`Storyshots basic/input/NumberInput Default 1`] = `
<form
  autoComplete="off"
  noValidate={true}
  onSubmit={[Function]}
>
  <div
    className="TextFieldWithCustomLabel__Wrapper-a48xik-0 jTvESA"
    onChange={[Function]}
  >
    <label
      className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-animated"
    >
      <div>
        Label
      </div>
    </label>
    <div
      className="MuiFormControl-root MuiTextField-root sc-fKgIGh ibXjNK"
    >
      <div
        className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-formControl MuiInputBase-adornedEnd MuiFilledInput-adornedEnd"
        onClick={[Function]}
      >
        <input
          aria-invalid={false}
          autoFocus={false}
          className="MuiInputBase-input MuiFilledInput-input MuiInputBase-inputAdornedEnd MuiFilledInput-inputAdornedEnd"
          disabled={false}
          onAnimationStart={[Function]}
          onBlur={[Function]}
          onChange={[Function]}
          onFocus={[Function]}
          required={false}
          type="text"
        />
        <div
          className="MuiInputAdornment-root MuiInputAdornment-filled MuiInputAdornment-positionEnd"
        >
          <span>
            Bla
          </span>
        </div>
      </div>
    </div>
  </div>
</form>
`;
